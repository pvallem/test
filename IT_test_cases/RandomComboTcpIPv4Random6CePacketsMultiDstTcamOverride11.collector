bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002d"
    dst_address: "\004\005\006\007\010\000"
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: MAC
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002e"
    dst_address: "\004\005\006\007\010\001"
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: MAC
  }
  reverse_flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "5\207\232\201"
    dst_address: "\313s#^"
    src_port: 13291
    dst_port: 11268
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\274`eY"
    dst_address: "\304f3r"
    src_port: 34058
    dst_port: 6526
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\031\207\215\'"
    dst_address: "\305>\214\030"
    src_port: 54336
    dst_port: 18562
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
