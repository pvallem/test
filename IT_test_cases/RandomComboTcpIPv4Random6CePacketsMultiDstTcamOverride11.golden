flow_info {
  key {
    src_address: "\031\207\215\'"
    dst_address: "\305>\214\030"
    src_port: 54336
    dst_port: 18562
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\004\005\006\007\010\000"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\304f3r"
    dst_address: "\274`eY"
    src_port: 6526
    dst_port: 34058
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\004\005\006\007\010\001"
    dst_address: "\002\002\002\002\002e"
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "5\207\232\201"
    dst_address: "\313s#^"
    src_port: 13291
    dst_port: 11268
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
